### PR TITLE
Added home

### PR DESCRIPTION
**Pull request by telupreethi17**
review and merge

### CODE REVIEW

File: SeleniumGrid/src/test/java/ptc/PreeHomee.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package ptc;
+
+public class PreeHomee {
+	public static void main(String[] args) {
+		System.out.println("Preethiii");
+	}
```
Comment (Kavu-23) [inline SeleniumGrid/src/test/java/ptc/PreeHomee.java:7]: add some more comments

**Review by Kavu-23 — COMMENTED** [review]
add changes

**Comment by telupreethi17** [comment]
i have doone

File: SeleniumGrid/src/test/java/ptc/PreeHomee.java (hunk 1 above)
Comment (Kavu-23) [reply SeleniumGrid/src/test/java/ptc/PreeHomee.java:7]: reviewed and merging

**Review by Kavu-23 — COMMENTED** [review]
merging

**Comment by Kavu-23** [comment]
reviewed and merged

**Pull request merged**